proto/v1/project_service.proto

```diff
@@ -267,4 +297,111 @@ enum ProjectRole {
   PROJECT_ROLE_UNSPECIFIED = 0;
   PROJECT_ROLE_OWNER = 1;
   PROJECT_ROLE_DEVELOPER = 2;
-}
\ No newline at end of file
+}
+
+message GetProjectIssueRequest {
+  // The name of the issue to retrieve.
+  // Format: projects/{project}/issues/{issue}
+  string name = 1 [(google.api.field_behavior) = REQUIRED];
+}
+
+message ListProjectIssuesRequest {
+  // The parent, which owns this collection of issues.
+  // Format: projects/{project}
+  // Use "projects/-" to list all issues from all projects.
```

Comment:
is this a standard practice to denote all?

e.g. why not use `*` or `projects/*`

Reply:
Yes, it's part of https://google.aip.dev/159.